src/watertap_contrib/reflo/unit_models/zero_order/tests/test_forward_osmosis.py

```diff
@@ -0,0 +1,272 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pytest
+import re
+from pyomo.environ import (
+    ConcreteModel,
+    value,
+    assert_optimal_termination,
+    units as pyunits,
+)
+from pyomo.network import Port
+from idaes.core import FlowsheetBlock, UnitModelCostingBlock
+from idaes.core.util.testing import initialization_tester
+from idaes.core.util.exceptions import ConfigurationError
+from idaes.core.util.model_statistics import (
+    degrees_of_freedom,
+    number_variables,
+    number_total_constraints,
+    number_unused_variables,
+    unused_variables_set,
+)
+from idaes.core.util.testing import initialization_tester
+from idaes.core.util.scaling import (
+    calculate_scaling_factors,
+    constraint_scaling_transform,
+    unscaled_variables_generator,
+    unscaled_constraints_generator,
+    badly_scaled_var_generator,
+)
+
+import idaes.logger as idaeslog
+
+from watertap.core.solvers import get_solver
+from watertap.property_models.seawater_prop_pack import SeawaterParameterBlock
+from watertap_contrib.reflo.unit_models.zero_order.forward_osmosis_zo import (
+    ForwardOsmosisZO,
+)
+from watertap_contrib.reflo.property_models.fo_draw_solution_properties import (
+    FODrawSolutionParameterBlock,
+)
+from watertap_contrib.reflo.costing import REFLOCosting
+
```

Comment:
I see there is a costing test in the Trevi flowsheet test, so I am ok with there not being one here.